index.rst

```diff
@@ -264,10 +265,10 @@ A notebook examining data should be deployment invariant within LSST operations;
 
 Units should be SI units, and the time stamps should be in TAI.
 
-.. .. rubric:: References
+References
+----------
 
-.. Make in-text citations with: :cite:`bibkey`.
+.. bibliography:: local.bib lsstbib/books.bib lsstbib/lsst.bib lsstbib/lsst-dm.bib lsstbib/refs.bib lsstbib/refs_ads.bib
+   :encoding: latex+latin
```

Comment:
Turns out the key is that this encoding field need to be deleted. Not entirely sure why.

```
.. bibliography:: local.bib lsstbib/books.bib lsstbib/lsst.bib lsstbib/lsst-dm.bib lsstbib/refs.bib lsstbib/refs_ads.bib
   :style: lsst_aa
```